ui/src/app/components/sharded-bulk-source-details-form/sharded-bulk-source-details-form.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="connect-load-database-container">
+    <div class="form-container">
+        <form [formGroup]="directConnectForm" class="shard-bulk-form">
+            <h4 class="primary-header">Add Data Shard Connection Details</h4>
+
+            <mat-form-field appearance="outline">
+                <mat-label>Input Type</mat-label>
+                <mat-select formControlName="inputType">
+                    <mat-option *ngFor="let element of inputOptionsList" [value]="element.value">
+                        {{ element.displayName }}
+                    </mat-option>
+                </mat-select>
+            </mat-form-field>
+
+            <div class="textInput" *ngIf="(directConnectForm.value.inputType === 'text')">
```

Comment:
Let's specifically call out that the form and this page will synced in real time.

Reply:
Do you mean that whatever is added to the form will get automatically displayed as JSON when the user clicks the text tab?
This won't happen today..